.eslintrc.json

```diff
@@ -28,6 +28,7 @@
             "MethodDefinition": true,
             "ClassDeclaration": true
         }
-    }]
+    }],
+    "no-unused-vars": ["error", { "args": "none" }]
```

Comment:
#### Nitpick
- Why did you add this rule, it's not in the original Eslint specifications on the PT board

Reply:
Thanks for pointing that. I figured that Hound repeatedly warned against unused function arguments, which in most cases is not relevant. Rule was initially added to resolve eslint `no-unused-vars` report for unused function arguments, e.g Express request object, sequelize migration etc.

Reply:
You could have just applied the rule to that file instead of the whole codebase. @devdbrandy

Reply:
Yeah, that works too. I'll go ahead and make adjustments, thanks.

Reply:
Resolved